tests/plugins/application/test_serverless_app_plugin.py

```diff
@@ -253,9 +255,53 @@ def __init__(self, app_id="app_id", semver="1.3.5"):
 
 #     self.plugin.on_before_transform_resource(app_resources[0][0], 'AWS::Serverless::Application', app_resources[0][1].properties)
 
-# class TestServerlessAppPlugin_on_after_transform_template(TestCase):
 
-#     def setUp(self):
-#         self.plugin = SeverlessAppPlugin()
-
-# # TODO: test this lifecycle event
+class TestServerlessAppPlugin_on_after_transform_template(TestCase):
```

Comment:
Interesting we didn't have test for this block before :(

Thanks for adding them.